### PR TITLE
New syntax documentation

### PR DESCRIPTION
**Pull request by vany0114**
Hi @reisenberger 

I've updated the documentation according to the new syntax changes. BTW I'm very bad at wording, so feel free to change it if necessary.

### CODE REVIEW

**Comment by reisenberger** [comment]
Nice, @vany0114!  Will do a full read in the next few days!

File: README.md [hunk 1]
```diff
@@ -34,46 +34,118 @@ Simmy offers the following chaos-injection policies:
 
 |Policy| What does the policy do?|
 | ------------- |------------- |
-|**[Fault](#Inject-fault)**|Injects exceptions or substitute results, to fake faults in your system.|
+|**[Exception](#Inject-exception)**|Injects exceptions in your system.|
+|**[Result](#Inject-result)**|Substitute results to fake faults in your system.|
 |**[Latency](#Inject-latency)**|Injects latency into executions before the calls are made.|
 |**[Behavior](#Inject-behavior)**|Allows you to inject _any_ extra behaviour, before a call is placed. |
 
 # Usage
```
Comment (reisenberger) [inline README.md:44]: Should we give a full example here, before we give the syntax for the specific parts/cases?  (the syntax for parts might make more sense after the reader has seen a full example)

    // Following example causes the policy to throw SocketException with a probability of 5% if enabled
    var fault = new SocketException(errorCode: 10013);
    var chaosPolicy = MonkeyPolicy.InjectException(with =>
        with.Fault(fault)
            .InjectionRate(0.05)
            .Enabled()
        );

File: README.md [hunk 2]
```diff
@@ -34,46 +34,118 @@ Simmy offers the following chaos-injection policies:
...
+## Inject exception
 ```csharp
-var chaosPolicy = MonkeyPolicy.InjectFault(
-    Exception | Func<Context, CancellationToken, Exception> fault,
-    double | Func<Context, CancellationToken, double> injectionRate, 
-    Func<bool> | Func<Context, CancellationToken, bool> enabled 
-);
+var chaosPolicy = MonkeyPolicy.InjectException(Action<InjectOutcomeOptions<Exception>>);
+```
+
```
Comment (reisenberger) [inline README.md:57]: Wondering if we should give an example here as well as the 'abstract' syntax? Maybe something like below:

((( Maybe we will end up moving all the examples from **Basic examples Step 1 Set up the Monkey Policy**. Maybe we could just merge that whole section into here - what do you think? )))

    var chaosPolicy = MonkeyPolicy.InjectException(Action<InjectOutcomeOptions<Exception>>);

> For example

    var chaosPolicy = MonkeyPolicy.InjectException(with =>
        with.Fault(new SocketException(errorCode: 10013))
            .InjectionRate(0.05)
            .Enabled()
        );

**Review by reisenberger — COMMENTED** [review]
@vany0114 This is great!  Main thought: maybe we should give full syntax examples before / alongside each abstract definition

    var chaosPolicy = MonkeyPolicy.InjectException(Action<InjectOutcomeOptions<Exception>>);

or

    PolicyOptions.InjectionRate(Double);

What do you think? Maybe it's just merging the **Step 1** examples section upwards into the Usage section, or starting with examples first

**Comment by vany0114** [comment]
@reisenberger I agree, I think it makes more sense in order for the user to understand the syntax/usage seeing the examples alongside each abstract definition. I merged the **Basic examples Step 1 Set up the Monkey Policy** section into the usage one as you suggested.

Thanks for the suggestions and Merry Christmas! 🎉

**Comment by vany0114** [comment]
By the way, feel free to polish the documentation if I missed something.

**Pull request merged**